docs/source/testing_with_ansibleee.rst

```diff
@@ -96,14 +96,14 @@ Create an NFS PV, and a PVC that can be mounted on the ansibleee pods.
 
     % oc apply -f edpm-ansible-storage.yaml
 
-Add extraMount to your OpenStackDataPlane CR
+Add extraMount to your OpenStackDataPlaneNodeSet CR
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Looks good, but please extend the `~~~~~~~` line to fit the new length because it failed with:
```
Validating...
./docs/source/testing_with_ansibleee.rst:100: D000 Title underline too short.
```
https://github.com/openstack-k8s-operators/edpm-ansible/actions/runs/6644350582/job/18053343343?pr=479#step:5:298